## This is the color description for each object
## By default, CSD is used.
## Related pkg: visor
##
## Author: Ferdian Pratama

Header header
int16 size # DEFAULT= 64 (VECTOR SIZE)
int16[] desc # CSD is used